content/desktop/use-desktop/builds.md

```diff
@@ -41,33 +26,104 @@ active builders. The list doesn't include builds for inactive builders.
 After the restart, the **Builds** view and the **Builders** settings menu
 appear.
 
+## Show build list
+
+Select the **Builds** view in the Docker Dashboard to open the build list.
+
+The build list shows running and completed builds for your active builders. The
+top-right corner shows the name of your currently selected builder, and the
+**Builder settings** button lets you [manage builders](#manage-builders) and in
+the Docker Desktop settings.
+
+Running builds are displayed in the top section of this view, with information
+about the build target and progress.
+
+The lower section the view shows your recent builds. You can select any of the
+builds from the active builds or completed builds section to view detailed
+information, such as logs, a performance breakdown, cache utilization, and
+more.
+
+The build list doesn't include builds for inactive builders. That is, builders
+that you've removed from your system, or builders that have been stopped.
+
 ## Inspect a build
 
 To inspect a build, select the build that you want to view in the list.
+The inspection view contains a number of tabs:
+
+### Info tab
+
+The **Info** tab displays details about the build.
```

Comment:
should we mention what the platform selector does here in the info tab?

Reply:
yeah good idea